conductor/clarisse_utils.py

```diff
@@ -103,24 +103,24 @@ def do_export():
     scene_info = {"output_path": ""}
     for i in range(attrs.get_count()):
         # deduplicating
-        file = attrs[i].get_string()
-        if not os.path.isfile(file):
+        _file = attrs[i].get_string()

```

Comment:
`file_path` would be document what this variable contains, because `_file` sounds to me more like a file handle than a path.

Reply:
This comment applies to other occurrences of the `_file` variable in this diff.